modules/apps/frontend-theme/frontend-theme-admin/src/css/components/_group-item.scss

```diff
@@ -83,3 +83,25 @@
 	padding-left: 10px;
 	padding-right: 4px;
 }
+
+.portlet-system-settings {
```

Comment:
Hey @pat270 , this looks good but we need this to apply to instance settings and site settings too. It looks like `list-group-card-item` is currently only used in our settings pages, so I think it's probably safe to just remove the `.portlet-system-settings` so that your styles apply to all `list-group-card-item`'s. What do you think?

Reply:
@ethib137 Thanks for pointing that out. I thought it was in Clay, but it might be left overs from 1.x. I removed some `.controls-visible.open` styles because there is no Product Menu on those pages.

My changes here apply to both System and Instance Settings. I'll post a screenshot at the top.

Reply:
@pat270 What about `Site Settings`? Just double checking.

Reply:
Just checked... it does apply to site settings as well.